operator/src/api.js

```diff
@@ -38,6 +37,7 @@ class OperatorApi {
         this.txOpts = {
             gas: 100712388,
             gasPrice: process.env.GAS_PRICE,
+            from: web3.eth.defaultAccount,
```

Comment:
I don't know why but not setting this results in an error: `Error: No "from" address specified in neither the given options, nor the default options.` where running the unit tests (see CI output).

Reply:
Turned out the Contracts don't inherit configuration from the web3 instance. I now pass this field to the Contract when instantiating it.